### PR TITLE
Updated the package version in the notebooks

### PR DESCRIPTION
**Pull request by Ark-kun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/1117)

### CODE REVIEW

**Comment by gaoning777** [comment]
would you mind also updating the version : https://github.com/kubeflow/pipelines/blob/master/samples/notebooks/DSL%20Static%20Type%20Checking.ipynb

**Comment by Ark-kun** [comment]
> would you mind also updating the version : https://github.com/kubeflow/pipelines/blob/master/samples/notebooks/DSL%20Static%20Type%20Checking.ipynb

Done. Missed that one as it used `kfp-experimental.tar.gz`

File: samples/tfx-oss/TFX Example.ipynb [hunk 1]
```diff
@@ -18,7 +18,7 @@
    "outputs": [],
    "source": [
     "!pip3 install https://storage.googleapis.com/ml-pipeline/tfx/tfx-0.12.0rc0-py2.py3-none-any.whl \n",
-    "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.10/kfp.tar.gz --upgrade\n"
+    "!pip3 install https://storage.googleapis.com/ml-pipeline/release/0.1.16/kfp.tar.gz --upgrade\n"
```
Comment (fabito) [inline samples/tfx-oss/TFX Example.ipynb]: `kfp-0.1.16` isn't compatible with `tfx-0.12.0` due to change in `add_pipeline` signature

Comment (Ark-kun) [reply samples/tfx-oss/TFX Example.ipynb]: Thanks for the catch!
We've fixed the issue in v0.1.17 and will update the samples soon.
We're also fixing TFX: https://github.com/tensorflow/tfx/pull/39

**Pull request merged**